test/libsolidity/syntaxTests/literalSuffixes/parameters/overloading_integer_vs_integer.sol

```diff
@@ -0,0 +1,21 @@
+function uSuffix(uint8) pure suffix returns (uint) {}
+function uSuffix(uint16) pure suffix returns (int) {}
+
+function iSuffix(int8) pure suffix returns (uint) {}
```

Comment:
hmm.. maybe I misunderstood somehow the documentation.. but with respect to ["There are no negative number literals in Solidity." ](https://github.com/ethereum/solidity/pull/14121/files#diff-85812ff9484018acfc04b0dc727b772bded418bb644ba7e1286b59e20d481481R146). Wouldn't it make sense to only support unsigned integers?

Reply:
See my comment on this in the docs PR. Overall this was a design decision with some reasons behind it and if you want it changed, it's probably better to bring it up on the operators channel because here the discussion is getting scattered in comments on multiple PRs and hard to follow.